### DISCUSSION TITLE
youtube_tutorial

### DISCUSSION THREAD

**Discussion by betatestsm2020** [rokath/trice#369] [Q&A]
Dear rokath,

This project looks great.

To be honnest, I just spent 30min working on it, witout success but it is late, maybe I am too sleepy xD.

May I propose to made a youtube tutorial to show how to add TRICE to a project + how to use it ?

For that I will sure need some of your advices and time for review.

Does it make sense to you ? 

Looking forward your answer.

**Comment by rokath** [comment]
If you, @betatestsm2020,  intend to do a Youtube video, it is a pleasure for me to help with that. Currently I am working on the next release (branch noids), but it will take some time, because it needs more testing effort. For now I recommend to use a diff tool and compare the 2 F030 test projects in the main branch and the noids branch. Use the stack buffer to get started.  It needs just a write function based for example on putchar.

**Reply by betatestsm2020** [reply]
Hi, 
There is no hurry, just to mention I will be glad to do some kind of how to use TRICE tutorial video.
I am also very busy with other projects, we can wait next major release to do so. 
You are the boss, just notify me when you think we can start something.
Regards

**Reply by breakersun** [reply]
taking a few days to wrapping my head, but still no clue.
Is there anything special in the MDK project settings besides the c files modifications?
I noticed trice.exe u and trice.exe z steps laying down the MDK project files. Sorry asking this cause I can only use make file project instead of Keil MDK for some legal reasons.

**Reply by rokath** [reply]
Makefiles are perfect. Run `trice u` in a console with the right switches. It should update your source tree with IDs and enrich your ti.json file. You can start with an empty one. If ok, put your `trice u` as a prebuild step in your makefiles. `trice z` do not use for now.

**Comment by rokath** [comment]
Hi [betatestsm2020](https://github.com/betatestsm2020), branch `noids` is now merged into `master`. It needs a few more checks before doing a release.

**Comment by rokath** [comment]
See v0.56.3

**Reply by betatestsm2020** [reply]
Hi rokath, 

Let me look and start by myself first to try the v0.56.3, then in a couple of days when you are available we can set a visio meeting to exchange on what to put on the youtube tutorial and discuss about it.

Regards

**Comment by rokath** [comment]
I guess, the most interested theme could be porting trice to a new hardware.

**Reply by betatestsm2020** [reply]
Could not be more agree on that, actually I spent 2 hours yesterday and a bit today, I think we can set a meeting if you agree.

**Comment by rokath** [comment]
I am living in Berlin,  Germany. Just make a proposal and I will let you know if it is ok.

**Reply by betatestsm2020** [reply]
I work for myself, so I have no contraint, what about Thursday or Friday morning ?

**Comment by rokath** [comment]
6.3. or 7.3. around 11 CET are fine.

**Reply by betatestsm2020** [reply]
Great, let 's go with 6.3 11 CET can I use th@seerose.net to send you the link ?
Are you ok with skype or we can use Jitsi, or any ?

**Comment by rokath** [comment]
I have neither. Please just send me the link to th@seerose.net and info and I will install whatever is needed. Looking forward to Monday. Probably I have an old skype account.

**Comment by rokath** [comment]
Good morning @betatestsm2020, until now no mail with a link arrived.

**Reply by betatestsm2020** [reply]
Ooo, i sent it saturday

**Reply by betatestsm2020** [reply]
I just resent it

**Comment by betatestsm2020** [comment]
here the link
https://meet.jit.si/Trice_log_meeting
